pkg/controllers/elasticsearch/cluster_control.go

```diff
@@ -121,12 +136,197 @@ func (e *defaultElasticsearchClusterControl) Sync(c *v1alpha1.ElasticsearchClust
 	}
 
 	// TODO: handle status
-	if err = e.nodePoolControl.Sync(c); err != nil {
+	if c.Status, err = e.configMapControl.Sync(c); err != nil {
+		e.recorder.Eventf(c, apiv1.EventTypeWarning, errorSync, messageErrorSyncConfigMap, err.Error())
+		return c.Status, err
+	}
+
+	state := &controllers.State{
+		Clientset:          e.kubeClient,
+		NavigatorClientset: e.navigatorClient,
+		Recorder:           e.recorder,
+		StatefulSetLister:  e.statefulSetLister,
+		ConfigMapLister:    e.configMapLister,
```

Comment:
Do any of the actions need to list ConfigMaps in this branch? 
I can't find it used in any of the actions.

Reply:
Not right now - once this is merged I'll be incorporating it though.